### DISCUSSION TITLE
Is it posible to add a custom entity relation to AbpUser

### DISCUSSION THREAD

**Discussion by josussunt** [abpframework/abp#19764] [General]
Hi everyone,

I am starting with abp.io and I need to add a related entity to AbpUser. I found how to add a new property like string or int and I read also about adding extra property. 

Can anyone answer:

1. Is it possible? 
2. If yes, where can I find some code sample, doc, or something that help me? I have been searching the docs, the comunity and google but I found nothing!!

Thank you

**Comment by maliming** [comment]
hi

You can refer to this question: https://support.abp.io/QA/Questions/2151/Add-relation-between-AbpUser-to-My-Table-in-50

**Comment by josussunt** [comment]
Thank you!! I will check it
